spec/features/two_factor_authentication/multiple_mfa_sign_up_spec.rb

```diff
@@ -45,7 +43,7 @@
     end
 
     scenario 'user can select 2 MFA methods and then chooses another method during' do
-      sign_in_before_2fa
+      sign_up_and_set_password
```

Comment:
I changed sign up method because the previous one failed because `in_account_creation_flow?` wasn't available so the click_link below couldn't find that link in the template.